### PR TITLE
Revert "Feature/Jupyterhub: Removed option to expose proxy svc using NLB"

### PR DESCRIPTION
**Pull request by shapirov103**
Reverts aws-quickstart/cdk-eks-blueprints#606

### CODE REVIEW

**Review by youngjeong46 — DISMISSED** [review]
Tested successfully and revised doc. good to go.

File: lib/addons/jupyterhub/index.ts [hunk 1]
```diff
@@ -14,6 +14,26 @@ import * as semver from 'semver';
 import { EbsCsiDriverAddOn } from "../ebs-csi-driver";
 import { EfsCsiDriverAddOn } from "../efs-csi-driver";
 
+/**
+ * Configuration options for exposing the JupyterHub proxy
+ */
+export enum jupyterHubServiceType {
```
Comment (shapirov103) [inline lib/addons/jupyterhub/index.ts:20]: Enum/classes/interface/types - start with upper case (affects code elsewhere and docs)

**Review by shapirov103 — COMMENTED** [review]
Minor comment

File: lib/addons/jupyterhub/index.ts (hunk 1 above)
Comment (shapirov103) [reply lib/addons/jupyterhub/index.ts:20]: > Enum/classes/interface/types - start with upper case (affects code elsewhere and docs)

@youngjeong46, looks like I am the one who submitted the PR and you are the reviewer. please disregard, I will address.

**Pull request merged**